addons/payment_mercado_pago/models/payment_provider.py

```diff
@@ -67,9 +67,13 @@ def _mercado_pago_make_request(self, endpoint, payload=None, method='POST'):
                     _logger.exception(
                         "Invalid API request at %s with data:\n%s", url, pprint.pformat(payload),
                     )
-                    response_content = response.json()
-                    error_code = response_content.get('error')
-                    error_message = response_content.get('message')
+                    try:
+                        response_content = response.json()
+                        error_code = response_content.get('error')
+                        error_message = response_content.get('message')
+                    except ValueError:
+                        error_code = response.status_code
+                        error_message = ''
```

Comment:
I think we should not try to force dummy error codes and messages into the existing ValidationError text; "information <empty> (code 403)" is useless to users.

Instead, we should raise a different ValidationError (`"The communication with the API failed. The response is empty. Please verify your access token."`) when a ValueError exception is raised.